src/damian_package/src/damian_node.cpp

```diff
@@ -0,0 +1,55 @@
+#include <rclcpp/rclcpp.hpp>
+#include <cv_bridge/cv_bridge.h>
+#include <opencv2/opencv.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+class DamianNode : public rclcpp::Node {
```

Comment:
Let's rename this node to TargetSelectionNode instead. Additionally, rename the folders and package name. The enclosing folder should be called target_selection, and the package name should also be target_selection